package.json

```diff
@@ -45,7 +45,6 @@
     "node": "18.x"
   },
   "scripts": {
-    "postinstall": "if ! command -v purs &> /dev/null; then npm install --no-save spago@0.21.0 purescript@0.15.10; fi",
```

Comment:
This just makes the nix build fail. can we remove it?

Reply:
@paluh ?

Reply:
The situation is as follows:

* We would like to be nix-agnostic meaning a contributor which doesn't use nix should be able to enter the project.
* In order to achieve that we want to preserve `package.json` -> `devDependencies` as complete as possible.
* We are not able to put in there dependencies which require binaries (like purescript complier) because this collided with our `nix develop && npm install`.
* This was an escape hatch - people without nix after entering the project and running `npm install` should still have `purescript` installed.

This probably changes after nixification as we should not use inside the project `npm install` to install dependencies but rather:

`npm install --package-lock-only`

and then somehow recreate the shell and reenter the dev env. Am I right @zeme-wana?

Reply:
Is there a way to tell 
```
webpack-cli --mode=production -c webpack.js
``` 
Not to run the `postinstall` script?